src/core/meshcop/leader.cpp

```diff
@@ -201,6 +201,22 @@ void Leader::HandleKeepAlive(Coap::Header &aHeader, Message &aMessage, const Ip6
     }
     else
     {
+        NetworkData::Leader &  netdata = GetNetif().GetNetworkDataLeader();
```

Comment:
nit: remove extra space between `&` and `netdata`

Reply:
clang-format aligns variable declarations in this way.